### DISCUSSION TITLE
What structure do we need? / Language?

### DISCUSSION THREAD

**Discussion by jwende** [open3e/open3e#53] [Wiki]
What language do we want to use - English oder Deutsch?

**Comment by philippoo66** [comment]
ich fürchte, die haben jetzt auf deutsch angefangen, oder?

ich denke, es ist besser was in deutsch zu haben, und ggf. auch unterschiedliche Sparachen, als nix. Es gibt heute googel translate, also sollte das kein Problem sein...

**Reply by TheSmartGerman** [reply]
Hallo,
ich habe auf Deutsch angefangen (ging einfach am schnellsten) -> könnte man auch "umschreiben":

https://github.com/TheSmartGerman/open3e/wiki

Ansonsten habe ich jetzt mal noch nach den Settings geschaut:

![image](https://github.com/abnoname/open3e/assets/9676982/0b0d7685-64b3-4635-adac-6061b885f5ec)

Wenn die Einstellung:
```
Restrict editing to collaborators only 
Public wikis will still be readable by everyone.`
```
raus nimmt, sollte jeder dokumentieren können.

**Reply by philippoo66** [reply]
die meisten Viessmann Anwender sind eh deutsch und einge davon fragen auch nach deutschem Text weil nicht jeder englisch so gut versteht.

Gibt's bei den Settings nicht auch noch einen Haken dass Änderungen zumindest von dir ge-reviewed werden, dass da nicht jemand völligen Blödsinn reinschreibt?

**Reply by jwende** [reply]
Die settings kann nur der owner anpassen … @abnoname was meinst du dazu? Oder sollten wir nach Alternativen suchen?

**Reply by rechi2** [reply]
Hab grad mal reingeschaut. Sieht doch klasse aus. Wo kann man beitragen? Hier oder bei TheSmartGerman?

**Reply by jwende** [reply]
Sind wir gerade am diskutieren - logisch gehört es hierher - der meiste Text steht schon bei @TheSmartGerman

**Reply by philippoo66** [reply]
ich denke das kann man hinterher auch rüberholen? https://til.simonwillison.net/github/migrate-github-wiki

ich hab das Gefühl, auf abnoname sind sowieso viele grade etwas 'müde', von daher ist es gut, wenn das Wiki erstmal mit frischen Kräften entsteht, oder?!

**Reply by philippoo66** [reply]
bis wir es rüberholen sollten wir auf abnoname/open3e aber einen Verweis nach TheSmartGerman machen!

**Reply by jwende** [reply]
Das mache ich gleich

**Reply by TheSmartGerman** [reply]
Wir können gerne erst auch einemal bei "mir" dokumentieren. Damit man es rüber holen kann, sollte das Wikie bei @abnoname allerdings so bleiben wie es ist. Also nicht hier auch beginnen und dann muss mach mühevoll versuche es zu mergen.

Ich habe die Settings mal angepasst, jetzt sollte jeder der bei GitHub angemeldet ist, daran arbeiten können... Bitte mal testen. 

Ich habe mal bewusst mit vorangestellten Nummern Seiten angelegt, damit man diese Sortieren kann und ggf. kann man auch sagen ungerade Nummer sind Englisch / garde Deutsch.

**Reply by jwende** [reply]
Dann zitiere ich mal
„Das ist der Weg“
